game2048/dbconnection/server.js

```diff
@@ -1,6 +1,7 @@
 const express = require('express');
 const path = require('path');
 const { Client } = require('pg');
+const { saltHashPassword } = require('./bcrypt-password-hash/passwordHashing.cjs');
 
 const app = express();
 const port = 3001;
```

Comment:
bellow is a config, that changes per user. Please get it in a config file

Reply:
added config.json file where client is being configured